evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -10,20 +13,17 @@ use serde::Deserialize;
 use std::{
     env, fs,
     io::{BufRead, BufReader, Read, Write},
-    path::Path,
+    path::{Path, PathBuf},
     process::Command,
     str::FromStr,
 };
 
 fn ffi(args: &[String]) -> Result<Bytes, Bytes> {
-    let output = Command::new(&args[0])
-        .args(&args[1..])
-        .output()
-        .map_err(|err| err.to_string().encode())?
-        .stdout;
+    let output =
+        Command::new(&args[0]).args(&args[1..]).output().map_err(util::encode_error)?.stdout;
```

Comment:
just noticed this, we should set the current dir of this command to the project root as well, right?

Reply:
fixed